wondrous-app/components/Common/TaskViewModalAutocomplete/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Grid } from '@mui/material';
+import { SafeImage } from 'components/Common/Image';
+import DefaultUserImage from 'components/Common/Image/DefaultUserImage';
+import SearchIcon from 'components/Icons/search';
+import { Arrow } from 'components/Icons/sections';
+import { useState } from 'react';
+import palette from 'theme/palette';
+import { ArrowWrapper, Option, PaperComponent, StyledAutocomplete, StyledTextField } from './styles';
+
+const profilePictureStyle = {
+  display: 'flex',
+  width: '24px',
+  height: '24px',
+  borderRadius: '50%',
+};
+
+const ProfilePicture = ({ profilePicture }) => (
```

Comment:
we have a component components/Common/ProfilePictureHelpers can we reuse that here?

Reply:
`ProfilePictureHelpers` does not use `useNextImage`. @vefimov recommends using it.